src/models/room-state.ts

```diff
@@ -60,6 +60,8 @@ export interface IPowerLevelsContent {
     // eslint-disable-next-line camelcase
     state_default?: number;
     ban?: number;
+    historical?: number;
```

Comment:
This does not exist in the spec so equally should not be present here https://spec.matrix.org/v1.9/client-server-api/#mroompower_levels 

`invite` is though and shocked to see this missing for so long.

Reply:
you're right, and also the spec has `notifications` which is missing here, i'll add it in place of `historical`